### PR TITLE
Python command line autocomplete

### PR DESCRIPTION
**Pull request by szabi-luxonis**
Add support for argument autocomplete on TAB pressing, like any other app.
`python3 -m pip install -r requirements.txt`
`echo 'eval "$(register-python-argcomplete depthai.py)"' >> ~/.bashrc`

Only bash is supported for now.
On macos run depthai from bash: from terminal: `bash` then follow instructions.

When pressing tab it will list all available arguments, and all available options for that argument, for example:
`./depthai.py -cnn` TAB will list all supported CNN models.
`./depthai.py -cnn mobi` TAB will autocomplete to mobilenet-ssd

### CODE REVIEW

**Comment by Luxonis-Brandon** [comment]
Wow this will be super sweet to have!  Thanks for thinking of this and making it happen.  I'll try it out once I get out from under the bit of a rock I'm under!

File: depthai_helpers/cli_utils.py [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
+
 from enum import Enum
+import os
+import consts.resource_paths
+
+def get_immediate_subdirectories(a_dir):
+    return [name for name in os.listdir(a_dir)
+            if os.path.isdir(os.path.join(a_dir, name))]
+
+stream_choices = ("metaout", "previewout", "jpegout", "left", "right", "depth_sipp", "disparity", "depth_color_h", "meta_d2h", "object_tracker")
```
Comment (VanDavv) [inline depthai_helpers/cli_utils.py:18]: `video` stream should be listed here also?

**Comment by VanDavv** [comment]
Super nice feature and very easy to integrate, kudos @szabi-luxonis!

File: depthai_helpers/cli_utils.py (hunk 1 above)
Comment (VanDavv) [reply depthai_helpers/cli_utils.py:18]: Also, since #148 was merged, the stream names also changed

**Pull request merged**